src/libs/facets/dismoi.facet.ts

```diff
@@ -0,0 +1 @@
+export const facetName = 'DisMoi';
```

Comment:
I think this would be best kept as an env variable. 
As I understand you don't want to have too many env variables. However, one possible way to overcome this potential issue, could to create an env variable with a complex structure.
We already have some array for example. So might have a simple js object or a json one if you prefer.
Or would it make sense to create another config file? For example let's look at a silly example:
```
# [...]
THEME_PATH=lmem.theme.json
```

The file idea might be too much :D but think about the idea of a more complex data structure kept in env....

Reply:
I also noticed `WEBSITE_DOMAIN` should be configurable as well.

Reply:
Yet another point... some translation keys are too specific and should be kept "facet" agnostic  like for example :
`profiles:action.know_more_dismoi`

Reply:
Since our app code will be aware of the facets (for example we will have things like :
```tsx
return facet === 'lmel' ? <This /> : <That />
```
), I think it defeats the point of having it all as env vars … Our app will be aware of facet particularities anyway … don’t you think ?

Reply:
About the translations keys, it’s being worked on here : #1155